replay/src/main.rs

```diff
@@ -8,13 +8,12 @@ extern crate webrender_traits;
 
 use bincode::serde::deserialize;
 use byteorder::{LittleEndian, ReadBytesExt};
-use euclid::Size2D;
 use gleam::gl;
 use std::io::Read;
 use std::fs::File;
 use std::path::{Path, PathBuf};
 use std::env;
-use webrender_traits::{RenderApi, PipelineId};
+use webrender_traits::{RenderApi, PipelineId, LayerSize, DeviceUintSize};
```

Comment:
Having `DeviceIntRect`, `DeviceUintRect`, and just `DeviceRect` now, the first thing that comes to mind is that the latter is inconsistent. Would be more consistent if it was `DeviceFloatRect`. But really, it would be much simpler to just have `DeviceRect<i32>`, `DeviceRect<u32>`, and so forth - we'd still get the safe units but with somewhat clearer semantics.